packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js

```diff
@@ -563,6 +563,61 @@ const defaultConfigs = {
     uniswapAddress: "0xedf187890af846bd59f560827ebd2091c49b75df",
     twapLength: 7200,
     invertPrice: true
+  },
+  DIGGBTC: {
+    type: "expression",
+    // Note: lower-case variables are intermediate, upper-case are configured feeds.
+    expression: `
+      mean(DIGG_WBTC_SUSHI, DIGG_WBTC_UNI)
+    `,
+    lookback: 7200,
+    minTimeBetweenUpdates: 60,
+    twapLength: 1800,
+    priceFeedDecimals: 8,
+    customFeeds: {
+      DIGG_WBTC_SUSHI: { type: "uniswap", uniswapAddress: "0x9a13867048e01c663ce8ce2fe0cdae69ff9f35e3" },
+      DIGG_WBTC_UNI: { type: "uniswap", uniswapAddress: "0xe86204c4eddd2f70ee00ead6805f917671f56c52" }
+    }
+  },
+  DIGGETH: {
+    type: "expression",
+    // Note: lower-case variables are intermediate, upper-case are configured feeds.
+    expression: `
+      digg_wbtc = mean(DIGG_WBTC_SUSHI, DIGG_WBTC_UNI);
+      wbtc_eth = mean(WBTC_ETH_SUSHI, WBTC_ETH_UNI);
+      digg_wbtc * wbtc_eth
```

Comment:
Any reason we couldn't just reuse the above price feed definition?

This would allow us to drop 2 of the customFeeds below too.

```suggestion
      wbtc_eth = mean(WBTC_ETH_SUSHI, WBTC_ETH_UNI);
      DIGGBTC * wbtc_eth
```